app/templates/all_questions/en/cyp_r1_all_questions_en.html

```diff
@@ -729,4 +729,4 @@ <h4 class="govuk-heading-s">
 </div>
     </div>
 </div>
-{% endblock %}
+{% endblock content %}
```

Comment:
is it required to add this over here ? because previously we did not add it here

Reply:
yes, it is one of the linting errors, that we need to name the blocks.